src/test/java/io/javalin/openapi/utils.kt

```diff
@@ -10,9 +10,11 @@ import io.swagger.v3.oas.models.OpenAPI
 import io.swagger.v3.oas.models.info.Info
 import org.assertj.core.api.Assertions.assertThat
 
+
+val OPENAPI_OPTION_BASIC = OpenApiOptions(Info().title("Example").version("1.0.0"))
```

Comment:
This can lead to problems. As OpenApiOptions ist mutable, you can change the reference in one test and it is changed in another. It would be better if you would create a function to create the basic options, so a new reference is created for every test.

Reply:
Actually considering that you can just do Yaml::pretty I think I will just pull this one.